apps/bluetooth/js/transfer.js

```diff
@@ -194,13 +196,9 @@ navigator.mozL10n.once(function showPanel() {
        */
       if (blob.name) {
         // The blob has name, send the blob directly.
-        defaultAdapter.sendFile(targetDevice.address, blob);
-        var msg = 'blob is sending...';
-        debug(msg);
-        if (--numberOfTasks === 0) {
-          transferred();
-        }
-      } else {
+        msg = 'blob is sending with name...';
```

Comment:
For each of these `msg = '...'` lines, I'd just write `debug('...')` instead. That way the sendFile() call does not need to have any debugging in it and you don't need to pass the argument.